src/index.html

```diff
@@ -1,56 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <meta
-    name="description"
-    content="Your Energy: Get Body in shape, Stay healthy."
-  />
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="Your Energy: Get Body in shape, Stay healthy."
+    />
 
-  <link rel="preconnect" href="https://fonts.googleapis.com">
-  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link
-    href="https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,100..1000;1,9..40,100..1000&display=swap"
-    rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,100..1000;1,9..40,100..1000&display=swap"
+      rel="stylesheet"
+    />
 
-  <link
-    href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/2.0.0/modern-normalize.min.css"
-    rel="stylesheet"
-    crossorigin="anonymous"
-    referrerpolicy="no-referrer"
-  />
-  <link href="./css/common.css" rel="stylesheet" />
-  <title>Your Energy</title>
-  <link rel="icon" type="image/x-icon" href="./favicon.svg" />
-</head>
-<body>
-<header class="header">
-  <load src="./partials/header.html" />
-</header>
-<main>
-  <!-- Hero Section -->
-  <load src="./partials/hero.html" />
-  <!-- Exercises Section -->
-  <load src="./partials/exercises.html" />
-  <!-- Quote of the day -->
-  <load src="./partials/quote.html" />
-</main>
-<footer class="footer">
-  <load src="./partials/footer.html" />
-</footer>
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/2.0.0/modern-normalize.min.css"
+      rel="stylesheet"
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link href="./css/main.css" rel="stylesheet" />
+    <title>Your Energy</title>
+    <link rel="icon" type="image/x-icon" href="./favicon.svg" />
+  </head>
+  <body>
+    <header class="header">
```

Comment:
Why is the space changed? Is it formatted with prettier?

Reply:
It was formatted with prettier

Reply:
```
"INFO" - 20:25:28] Detected local configuration (i.e. .prettierrc or .editorconfig), VS Code configuration will not be used
["INFO" - 20:25:28] Prettier Options:
{
  "filepath": "/Users/study/code/advanced-js/goit-advancedjs-team-3/src/index.html",
  "parser": "html",
  "useTabs": false,
  "tabWidth": 2,
  "printWidth": 80,
  "semi": true,
  "singleQuote": true,
  "trailingComma": "es5",
  "bracketSpacing": true,
  "arrowParens": "avoid",
  "proseWrap": "always"
}
["INFO" - 20:25:28] Formatting completed in 9ms.
```

Reply:
In my local version of prettier file I see this:

```
{
  "printWidth": 80,
  "useTabs": false,
  "semi": true,
  "singleQuote": true,
  "trailingComma": "es5",
  "bracketSpacing": true,
  "arrowParens": "avoid",
  "proseWrap": "always"
}
```